src/style.css

```
@@ -0,0 +1,60 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

```suggestion
.stars__star {
  background-repeat: no-repeat;
  background-position: center;
  width: 16px;
  height: 16px;
  margin-right: 4px;
}
```
why do you need all this selectors?)

---

thank you! fixed)